pkg/agent/main.go

```diff
@@ -35,6 +36,10 @@ func main() {
 	if err != nil {
 		klog.Exitf("Error adding lighthouse V1 to the scheme: %v", err)
 	}
+	err = lighthousev2a1.AddToScheme(scheme.Scheme)
```

Comment:
This is something the generator does not do for some reason.

We need to register them in the register.go part, not here :)

https://github.com/submariner-io/submariner/blob/master/pkg/apis/submariner.io/v1/register.go#L28

@skitt also had a fun time about it.

Reply:
> This is something the generator does not do for some reason.
> 
> We need to register them in the register.go part, not here :)
> 
> https://github.com/submariner-io/submariner/blob/master/pkg/apis/submariner.io/v1/register.go#L28
> 
> @skitt also had a fun time about it.

That is already done:
https://github.com/submariner-io/lighthouse/blob/master/pkg/apis/lighthouse.submariner.io/v2alpha1/register.go#L31

Without this line I was getting an error at runtime. We had to add it for multiclusterservices too.

Reply:
Hmm, may be it was somewhere else @skitt do you remember?

thanks vishal